mavisp/modes.py

```diff
@@ -59,44 +59,55 @@ class MAVISpSimpleMode(MAVISpMode):
     'local_interactions_DNA', 'local_interactions_homodimers', 'sas', 'ptms',
     'denovo_phospho', 'long_range', 'functional_sites', 'clinvar', 'alphafold',
     'demask', 'gemme', 'eve', 'alphamissense', 'experimental_data']
-    supported_metadata = ['uniprot_ac', 'refseq_id', 'review_status', 'curators',
-    'gitbook_entry', 'allosigma_distance_cutoff']
-    index_cols = ['system', 'uniprot_ac', 'refseq_id', 'review_status', 'curators',
-    'gitbook_entry', 'allosigma_distance_cutoff' ]
-    index_col_labels = {'system' : "Protein",
-                        'uniprot_ac' : 'Uniprot AC',
-                        'refseq_id' : "RefSeq ID",
-                        'review_status' : 'Review status',
-                        'curators' : 'Curators',
-                        'gitbook_entry' : 'GitBook report',
-                        'allosigma_distance_cutoff' : 'Distance cut-off used for AlloSigma2'}
+    supported_metadata = ['uniprot_ac', 'refseq_id', 'review_status', 'curators', 'gitbook_entry',
+                          'allosigma_distance_cutoff', 'structure_source', 'structure_description',
+                          'linker_design', 'pdb_id']
+    index_cols = ['system', 'uniprot_ac', 'refseq_id', 'review_status', 'curators', 'gitbook_entry',
+                  'allosigma_distance_cutoff', 'structure_source', 'structure_description',
+                  'linker_design', 'pdb_id']
+    index_col_labels = {'system': "Protein",
+                        'uniprot_ac': 'Uniprot AC',
+                        'refseq_id': "RefSeq ID",
+                        'review_status': 'Review status',
+                        'curators': 'Curators',
+                        'gitbook_entry': 'GitBook report',
+                        'allosigma_distance_cutoff': 'Distance cut-off used for AlloSigma2',
+                        'structure_source': 'Structure source',
+                        'structure_description': 'Description of structure source',
+                        'linker_design': 'Linker design included',
+                        'pdb_id': 'PDB ID (if applicable)'}
+    structure_sources = {
+        "AFDB": "AlphaFold database (trimmed)",
+        "AF3": "AlphaFold3 webserver",
+        "AF2": "AlphaFold2 standalone",
+        "PDB": "Experimental PDB structure (trimmed or used)",
+        "Mod": "Homology model (PDB template, reconstruction)"}
```

Comment:
cam you change these to:

        "AFDB": "AlphaFold database",
        "AF3": "AlphaFold3 webserver",
        "AF2": "AlphaFold2 standalone",
        "PDB": "Experimental PDB structure",
        "Mod": "Homology model (PDB template, reconstruction)"}